src/app/pages/payment/international-payment-status/international-payment-status.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  selector: 'app-international-payment-status',
+  templateUrl: './international-payment-status.component.html',
+  styleUrls: ['./international-payment-status.component.scss']
+})
+export class InternationalPaymentStatusComponent implements OnInit {
+  imgSrc: any;
```

Comment:
`imgSrc` should be of type `string`. Type `any` can allow someone, later on, to change the value to something other than a string and typescript would still allow it, but it would end up breaking your implementation